lib/sqlalchemy/orm/session.py

```diff
@@ -59,6 +59,9 @@ class _SessionClassMethods(object):
     """Class-level methods for :class:`.Session`, :class:`.sessionmaker`."""
 
     @classmethod
+    @util.deprecated("1.3",
+                     "Calls to classmethod close_all are deprecated."
```

Comment:
**mike bayer** (zzzeek) wrote:

"""The :meth:`.Session.close_all` method is deprecated and be removed in a future release.  Please refer to :func:`.session.close_all`.
"""

note the RST markup is OK here it gets filtered out when the warning is emitted, the text is also added to the documentation.